components/admin/permissions/permission-group.tsx

```
@@ -0,0 +1,234 @@
+
+  return (
+    <div className={groupContainerClasses}>
+      {/* Group Header */}
+      <div className={groupHeaderClasses}>
+        <div className={groupTitleClasses}>
+          <button
+            onClick={onToggleExpanded}
+            className={toggleButtonClasses}
+            aria-expanded={isExpanded}
+            aria-label={`${isExpanded ? 'Collapse' : 'Expand'} ${group.name}`}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)